algorithms/kernel/objective_function/cross_entropy_loss/oneapi/cross_entropy_loss_dense_default_oneapi_impl.i

```diff
@@ -467,7 +452,7 @@ services::Status CrossEntropyLossKernelOneAPI<algorithmFPType, defaultDense>::co
         DAAL_CHECK_STATUS(status, dependentVariables->getBlockOfRows(0, nRows, ReadWriteMode::readOnly, yBlock));
 
         {
-            DAAL_ITTNOTIFY_SCOPED_TASK(compute.chunkData.getXY);
+            DAAL_ITTNOTIFY_SCOPED_TASK(getXY);
```

Comment:
Probably we should mark in the scoped task that this call of `getXY()` is related to cross-entropy. I guess it may be called in multiple places. When we write ` DAAL_ITTNOTIFY_SCOPED_TASK(compute.chunkData.getXY)`, we understand, when the call has happened